### DISCUSSION TITLE
does this also removes audio ads ?

### DISCUSSION THREAD

**Discussion by samyakbardiya** [Daksh777/SpotifyNoPremium#59] [Q&A] [answered]
I am trying to find something like [spotify-adblock](https://github.com/abba23/spotify-adblock) or [BlockTheSpot](https://github.com/mrpond/BlockTheSpot) but for MacOS, so was curious will this also remove the audio ads or it this just for UI ads ?

**Comment by Daksh777** [comment, marked as answer]
Yes, it removes audio ads as well

**Reply by samyakbardiya** [reply]
thanks for the quick response :D
